debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/ConnectionContext.java

```diff
@@ -132,8 +142,12 @@ public String hosts() {
         return config.getString(MongoDbConnectorConfig.HOSTS);
     }
 
-    public int pollPeriodInSeconds() {
-        return config.getInteger(MongoDbConnectorConfig.POLL_INTERVAL_SEC);
+    public Duration pollInterval() {
+        if (config.hasKey(MongoDbConnectorConfig.POLL_INTERVAL_SEC.name())) {
```

Comment:
I think we gave the new option precedence in case both are given in similar cases?

Reply:
I can definitely change it to be consistent.

Reply:
+1